group_vars/all/mails.yml

```diff
@@ -0,0 +1,27 @@
+---
+##
+# Mailjet smtp configuration (authentication in mails-secrets.yml)
+##
+mailjet__smtp_host: in-v3.mailjet.com
+mailjet__stmp_port: 587
+mailjet__smtp_tls: true
+
+##
+# nullmailer default configuration (using mailjet)
+##
+# Send all mails though mailjet only
+nullmailer__remotes:
+  - >
+    {{ mailjet__smtp_host }}
+    smtp --port={{ mailjet__stmp_port }}
+    {% if mailjet__smtp_tls %}--starttls{% endif %}
+    --user={{ mailjet__api_key }}
+    --pass={{ mailjet__secret_key }}
+
+# All mails to <user>@localhost or <user>@<nullmailer__mailname>
+# are redirected to this email (ie. no local mailbox)
+nullmailer__admin: infra@openfoodfacts.org
```

Comment:
why not root@openfoodfacts.org ?

Reply:
I'd love it if we could use an address other than root for those emails. E-mails addressed to root@<default_domain> often flags that some services might not be set up correctly. This small change will help us easily distinguish between mail sent by servers that are properly configured and relay via MailJet and mail sent by our legacy servers still using their own MTAs.

Reply:
ok, good idea.